modules/smart-agent_system-common/detectors-system.tf

```diff
@@ -23,5 +22,7 @@ EOF
     parameterized_subject = var.message_subject == "" ? local.rule_subject_novalue : var.message_subject
     parameterized_body    = var.message_body == "" ? local.rule_body : var.message_body
   }
+
+  max_delay = var.disk_running_out_max_delay
```

Comment:
Not directly related to this PR, is it intentional to have no heartbeat detector for system disk space metrics?